src/app/containers/ArticleFigure/__snapshots__/index.test.jsx.snap

```diff
@@ -92,40 +86,25 @@ exports[`ArticleFigure should render a lazyloaded image when lazyLoad set to tru
 }
 
 <figure
-  className="c0"
+  class="c0"
 >
   <div
-    className="c1"
+    class="c1"
   >
     <div
-      className="c2"
+      class="c2"
     >
       <div
-        className="c3"
+        class="c3"
       >
         <div
-          className="lazyload-placeholder"
-          style={
-            Object {
-              "height": undefined,
-            }
-          }
+          class="lazyload-placeholder"
```

Comment:
Can you add assertion tests for `ImageWithPlaceholder` to make up for the loss in snap data for this please?